test/TankTests.cpp

```diff
@@ -105,6 +112,14 @@ TEST_F(TankTest, WhenTankIsOneAxisOutOfTheBoard_ThenShouldAllowToMoveOnlyOneAxis
   expect_vec2f_eq({0.f, 500.f}, mTankSUT.get_position());
 }
 
+TEST_F(TankTest, WhenTankTakesHitForZeroDamage_ThenShouldBeDestroyed) {
+  EXPECT_FALSE(mTankSUT.take_hit(0));
+}
+
+TEST_F(TankTest, WhenTankTakesHitForWholeHealth_ThenShouldBeDestroyed) {
```

Comment:
Maybe one more test would be a good idea when tank takes multiple hits? To check if the damage sums up? 2 hits for half health should destroy the tank, right?

Reply:
Good point, I added another test case